package/MDAnalysis/coordinates/XDR.py

```diff
@@ -116,8 +116,12 @@ def _read_offsets(self, store=False):
                 np.savez(offsets_filename(self.filename),
                          offsets=offsets, size=size, ctime=ctime)
             except Exception as e:
-                warnings.warn("Couldn't save offsets because: {}".format(
-                    e.message))
+                try:
```

Comment:
So coincidence I just triggered the exception under python 2 and got a warning that `BaseException.message` deprecated is. So I guess they removed in Python 3. But that means that the code in your except branch should work under python 2.7 as well.